config/config.js

```diff
@@ -17,5 +17,6 @@ module.exports = {
   hotjarId: process.env.HOTJAR_ANALYTICS_TRACKING_ID,
   googleAnalyticsId: process.env.GOOGLE_ANALYTICS_TRACKING_ID,
   webtrendsId: process.env.WEBTRENDS_ANALYTICS_TRACKING_ID,
-  cacheTimeoutSeconds: process.env.CACHE_TIMEOUT_SECONDS || 0
+  cacheTimeoutSeconds: process.env.CACHE_TIMEOUT_SECONDS || 0,
+  liveServiceHostname: process.env.LIVE_SERVICE_HOSTNAME || 'www.nhs.uk'
```

Comment:
I'm not sure the name of the variable is right. I can see the default value is for the live service but other environments e.g. integration are not live. Or, if this is meant to mean live as in the default service people are going to, that isn't always going to be the case. It might be confusing in the future as to what it should be i.e. live or default. Hopefully that makes some sense.

Reply:
Since it is going to be the whole login link I'll be renaming it.